applications/StructuralMechanicsApplication/custom_elements/prestress_membrane_element.cpp

```diff
@@ -1307,8 +1307,8 @@ void PrestressMembraneElement::ComputePrestress(const unsigned int& rIntegration
     this->SetValue(PRESTRESS_AXIS_1, prestress_direction1);
     this->SetValue(PRESTRESS_AXIS_2, prestress_direction2);
 
-    if(GetProperties().Has(PROJECTION_TYPE_COMBO)){
-        if(GetProperties()[PROJECTION_TYPE_COMBO] != "file"){
+    // if(GetProperties().Has(PROJECTION_TYPE_COMBO)){
```

Comment:
What is PROJECTION_TYPE_COMBO?

The solution looks like a shortcut, maybe we should tackle PROJECTION_TYPE_COMBO lack of definition and throw an error in case not defined

Reply:
In fact there is a call for PROJECTION_TYPE_COMBO later, so this will not solve the problem, or should not

Reply:
just to say that also one should NEVER do string comparison within an element